modules/standard/Version.chpl

```diff
@@ -68,57 +120,20 @@ module Version {
     corresponds to version ``0.x.y``/``1.x.y`` in traditional semantic
     versioning.
   */
-
   const chplVersion;
-  chplVersion = createVersion(chplMajor, chplMinor, chplUpdate, chplSHA);
-
-
-  /*
-    A helper function that creates a new version value from its
-    arguments.
-
-    :arg major: The major version number
-    :type major: `int`
-
-    :arg minor: The minor version number
-    :type minor: `int`
-
-    :arg update: The optional update version number (defaults to 0)
-    :type update: `int`
-
-    :arg commit: The optional commit ID (defaults to "")
-    :type commit: `string`
-
-    :returns: A new version value of type :type:`sourceVersion`.
-  */
-  proc createVersion(param major: int,
-                     param minor: int,
-                     param update: int = 0,
-                     param commit: string = ""): sourceVersion(?) {
-    return new sourceVersion(major, minor, update, commit);
-
-  }
-
+  chplVersion = new versionValue(chplMajor, chplMinor, chplUpdate, chplSHA);
 
   /*
     This record represents a software version in a Git repository.  It
```

Comment:
With the understanding it was here before this PR, why is this specific to a Git repository?